.pre-commit-config.yaml

```diff
@@ -601,7 +601,7 @@ repos:
           - --max-length=70
         language: python
         files: ^\.pre-commit-config\.yaml$
-        additional_dependencies: ['pyyaml']
+        additional_dependencies: ['pyyaml', 'jinja2', 'black']
```

Comment:
In files above you can also add `|^scripts/ci/pre_commit/pre_commit_check_pre_commit_hook_names.py$`  - this way pre-commit will also be automatically run when the "pre-commit script" itself is changed as well - not only when .yaml file changes.